kaznet/settings/common.py

```diff
@@ -181,7 +181,7 @@
 ONA_BASE_URL = 'https://stage-api.ona.io'
 ONA_CROSS_AUTHENTICATION_URL = 'https://stage-api.ona.io/api/v1/user'
 ONA_USERNAME = 'kaznettest'
-ONA_ORG_NAME = 'kaznettest'
+ONA_ORG_NAME = 'onasystemsinc'
```

Comment:
Don't make this change here please.  Instead, use `override_settings`.  Look at any of the other tests for examples